src/OpenSheetMusicDisplay/OpenSheetMusicDisplay.ts

```diff
@@ -13,14 +13,20 @@ import {MXLHelper} from "../Common/FileIO/Mxl";
 import {Promise} from "es6-promise";
 import {AJAX} from "./AJAX";
 import * as log from "loglevel";
+import { DrawingParametersEnum, DrawingParameters } from "../MusicalScore/Graphical/DrawingParameters";
 
 export class OpenSheetMusicDisplay {
     /**
      * The easy way of displaying a MusicXML sheet music file
      * @param container is either the ID, or the actual "div" element which will host the music sheet
      * @autoResize automatically resize the sheet to full page width on window resize
      */
-    constructor(container: string|HTMLElement, autoResize: boolean = false, backend: string = "svg") {
+    constructor(container: string|HTMLElement,
+                options: OSMDOptions = {
+                    autoResize: false,
```

Comment:
I'd leave autoResize to true

Reply:
it was false before, but we can talk about it ;)

Reply:
I know, but I think that was a mistake.

Reply:
Yes, autoResize true is probably better default, changed in next commit.